syntax = "proto2";
package smartmouse.msgs;

/// \brief Message Gui Actions

message GuiActions
{
    /// \brief Action for Help > Source Code
    required bool source_code_action = 1;
}